ttyclock.c

```diff
@@ -523,6 +564,21 @@ key_event(void)
           set_box(!ttyclock->option.box);
           break;
 
+     case '"':
+     case '!':
+     case '@':
+     case '#':
+     case '$':
+     case '%':
+     case '^':
+     case '&':
+     case '*':
+     case '(':
```

Comment:
we do we use shift here? this is bound to fail depending on the keyboard mapping... why not just use the normal digits?